src/trix/elements/trix_editor_element.coffee

```diff
@@ -196,6 +196,7 @@ Trix.registerElement "trix-editor", do ->
       @editorController.registerSelectionManager()
       @registerResetListener()
       @registerClickListener()
+      @registerWithLabels()
```

Comment:
I haven't quite figured out how to _unhook_ these label associations when the `<label for="...">` changes, when the `<label>` element is no longer an ancestor of the `<trix-editor>`, or if the `<trix-editor>` element is removed.

Reply:
Investigate: https://web.dev/more-capable-form-controls/#defining-a-form-associated-custom-element